app/policies/user_policy.rb

```diff
@@ -0,0 +1,13 @@
+class UserPolicy < ApplicationPolicy
+  def show?
+    true
+  end
+
+  def profile?
+    update?
+  end
```

Comment:
Maybe  `alias profile? update?`

Reply:
🤔 is a possibility but they are not conceptually alias imo